examples/grpc-js/tracer.js

```diff
@@ -9,13 +10,26 @@ const { ZipkinExporter } = require('@opentelemetry/exporter-zipkin');
 const EXPORTER = process.env.EXPORTER || '';
 
 module.exports = (serviceName) => {
-  const provider = new NodeTracerProvider({
-    plugins: {
-      '@grpc/grpc-js': {
-        enabled: true,
-        path: '@opentelemetry/plugin-grpc-js',
+  const provider = new NodeTracerProvider();
+  registerInstrumentations({
+    instrumentations: [
```

Comment:
i think its worth to wait for #1806 so we can move all core plugins to new instrumentations

Reply:
registerInstrumentations loads plugins and instrumentations, don't need to wait, the next ticket #1761 will wait until all plugins are moved